### PR TITLE
Chinese translate

### PR DESCRIPTION
**Pull request by zrll12**
add Chinese translation

### CODE REVIEW

**Comment by Dqu1J** [comment]
Thanks for contributing! This will be added in the next version.

**Comment by zrll12** [comment]
By the way, I do not find chicken nugget in both the wiki and the code. Has it been added into the  game so far?

**Comment by Dqu1J** [comment]
It's a easter egg item, that's why it isn't in the wiki yet. But it has been added already

> By the way, I do not find chicken nugget in both the wiki and the code. Has it been added into the game so far?

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  "item.additionaladditions.rose_gold_hoe": "玫瑰金锄",
+  "item.additionaladditions.rose_gold_helmet": "玫瑰金头盔",
+  "item.additionaladditions.rose_gold_chestplate": "玫瑰金胸甲",
+  "item.additionaladditions.rose_gold_leggings": "玫瑰金护腿",
+  "item.additionaladditions.rose_gold_boots": "玫瑰金靴子",
+  "item.additionaladditions.wrench": "扳手",
+  "item.additionaladditions.fried_egg": "煎鸡蛋",
+  "item.additionaladditions.trident_shard": "三叉戟碎片",
+  "item.additionaladditions.glow_stick": "荧光棒",
+  "item.additionaladditions.honeyed_apple": "蜂蜜苹果",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:18]: Should probably be `蜜饯苹果`:
The original translation is too literal, the suggested one is more "beautiful".

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  "item.additionaladditions.depth_meter": "海拔计量器",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:19]: Should probably be `深度计`:
No comment

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  "item.additionaladditions.gilded_netherite_leggings": "镶金的下界合金护腿",
+  "item.additionaladditions.gilded_netherite_boots": "镶金的下界合金靴子",
+  "item.additionaladditions.music_disc_0308": "音乐唱片",
+  "item.additionaladditions.music_disc_1007": "音乐唱片",
+  "item.additionaladditions.music_disc_1507": "音乐唱片",
+  "item.additionaladditions.music_disc_0308.desc": "adoghr - 0308",
+  "item.additionaladditions.music_disc_1007.desc": "adoghr - 1007",
+  "item.additionaladditions.music_disc_1507.desc": "adoghr - 1507",
+  "item.additionaladditions.mysterious_bundle": "神秘包裹",
+  "item.additionaladditions.pocket_jukebox": "便携式音乐播放器",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:36]: Should probably be "袖珍唱片机":
The original translation doesn't match with Mojang's Jukebox translation; Too technical;

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  "item.minecraft.potion.effect.long_haste_potion": "加速药水",
+  "item.minecraft.potion.effect.strong_haste_potion": "加速药水",
+  "item.minecraft.splash_potion.effect.glow_potion": "喷溅型发光药水",
+  "item.minecraft.splash_potion.effect.long_glow_potion": "喷溅型发光药水",
+  "item.minecraft.splash_potion.effect.haste_potion": "喷溅型加速药水",
+  "item.minecraft.splash_potion.effect.long_haste_potion": "喷溅型加速药水",
+  "item.minecraft.splash_potion.effect.strong_haste_potion": "喷溅型加速药水",
+  "item.minecraft.lingering_potion.effect.glow_potion": "滞留型发光药水",
+  "item.minecraft.lingering_potion.effect.long_glow_potion": "滞留型发光药水",
+  "item.minecraft.lingering_potion.effect.haste_potion": "滞留型加速药水",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:59]: Should probably be `滞留型急迫药水`:
Original translation usually refers to speed

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  "item.minecraft.lingering_potion.effect.long_haste_potion": "滞留型加速药水",
+  "item.minecraft.lingering_potion.effect.strong_haste_potion": "滞留型加速药水",
+  "item.minecraft.tipped_arrow.effect.glow_potion": "发光箭",
+  "item.minecraft.tipped_arrow.effect.long_glow_potion": "发光箭",
+  "item.minecraft.tipped_arrow.effect.haste_potion": "加速箭",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:64]: Should probably be `急迫箭`:
Original translation usually refers to speed

File: src/main/resources/assets/additionaladditions/lang/zh_ch.json [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  "additionaladditions.advancements.obtain_glow_stick.desc": "使用荧光棒点亮洞穴",
+  "additionaladditions.advancements.obtain_mysterious_bundle": "乱买一气",
+  "additionaladditions.advancements.obtain_mysterious_bundle.desc": "从旅行商人手中购买神秘包裹",
+  "additionaladditions.advancements.obtain_pocket_jukebox": "重金属",
+  "additionaladditions.advancements.obtain_pocket_jukebox.desc": "在旅途中使用便携式音乐播放器听音乐",
+  "additionaladditions.advancements.obtain_rose_gold": "玫瑰金",
+  "additionaladditions.advancements.obtain_rose_gold.desc": "打造玫瑰金装备",
+  "additionaladditions.advancements.obtain_scoped_crossbow": "狙击手",
+  "additionaladditions.advancements.obtain_scoped_crossbow.desc": "为弩加装瞄准镜",
+  "additionaladditions.advancements.shot_self_scoped_crossbow": "自相残杀",
```
Comment (Dqu1J) [inline src/main/resources/assets/additionaladditions/lang/zh_ch.json:84]: Should probably be `自作自受`:
Original translations refers to cannibalism, mutual self-harm

**Review by Dqu1J — CHANGES_REQUESTED** [review]
According to another person from CurseForge that wants to contribute Chinese translations, there are some errors in the current translations. Could you please verify that and update your translations? Thanks!

**Comment by Dqu1J** [comment]
Please, review the requested changes and update your current translations.

**Comment by zrll12** [comment]
I agree to these changes, they’re better. So what should I do to review them?

**Comment by Dqu1J** [comment]
You need to implement them (put to file, commit, push), and re-request review.

**Comment by Dqu1J** [comment]
If you have any questions related to a change, comment on that conversation. If not, close it after implementing the changes.

**Review by zrll12 — COMMENTED** [review]
Thanks for correcting me.

**Review by Dqu1J — APPROVED** [review]

**Pull request merged**